app/views/shared/_applicant_declaration.html.erb

```diff
@@ -1,6 +1,6 @@
 <p class="govuk-body"><%= t('.submission_list_summary') %>:</p>
 
-<ul class="govuk-list govuk-list--bullet">
+<ul class="govuk-list govuk-list--bullet" id="declaration-agree-list">
```

Comment:
Again - rather than adding an `id` I think a descriptive class would be far better. Something like `"show-bullet-on-print"` or `"print-fix-show-bullet"`

Reply:
Done